pkgs/development/compilers/vlang/default.nix

```diff
@@ -39,6 +47,18 @@ stdenv.mkDerivation rec {
 
   preBuild = ''
     export HOME=$(mktemp -d)
+    # We need to set the target system, because the auto-detection doesn't work that great.
+    export VFLAGS="-os ${targetSystem} -showcc -skip-unused"
+    # We also need gc.h in the include path.
+    export CFLAGS="-I${staticBoehmgc.dev}/include"
+    export LDFLAGS="-L${staticBoehmgc}/lib"
```

Comment:
Does adding the package to buildInputs work?

Reply:
Fixed in https://github.com/COLAMAroro/nixpkgs/blob/dcaed354bd52104677eb30460088e627ce74e465/pkgs/development/compilers/vlang/default.nix#L52